tardis/simulation/convergence_plot.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+
+class ConvergencePlot:
```

Comment:
This class seems to be able to do everything with static methods and there's no __init__.  Does this actually need to be a class or could the whole thing just be a module?  If it is a class, what attributes does it have and keep track of?

Reply:
@Rodot- 
Yeah we can refactor this class as a module with standalone functions. But, even though the current implementation of ConvergencePlot does not have any instance variables, it may be useful to keep it as a class if there is a possibility that instance variables may be added in the future to track and store the convergence data across multiple function calls.

Please let me know if there's any changes required!